test/main_test.go

```diff
@@ -60,12 +61,12 @@ func TestMain(m *testing.M) {
 	if err != nil {
 		panic(fmt.Errorf("Error parsing features: %v", err))
 	}
-	if framework.IsConfigShowScenarios() {
+	if config.IsShowScenarios() {
 		showScenarios(features)
 	}
 
-	if !framework.IsConfigDryRun() {
-		if matchingFeature(cliTag, features) {
+	if !config.IsDryRun() {
+		if !config.IsCrDeploymentOnly() && matchingFeature(cliTag, features) {
```

Comment:
Is matchingFeature(cliTag, features) needed any more?
Do we still use cli tag anywhere?

Reply:
you are right. will remove that ...
we still however need to check that cli binary is there if we are not `cr_deployment_only`

Reply:
in fact I just found that in features/olm, we have disabled test that are checking cli commands ...

Reply:
just set `||` instead of `&&` as we still have some cli tests